docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -5,11 +5,17 @@ This guide provides steps on creating a Docker image with Intel® Distribution o
 ## <a name="system-requirments"></a>System Requirements
 
 @sphinxdirective
-.. tab:: Target Operating Systems
-
-  * Ubuntu 18.04 long-term support (LTS), 64-bit
-  * Ubuntu 20.04 long-term support (LTS), 64-bit
-  * Red Hat Enterprise Linux 8, 64-bit
+.. tab:: Target Operating Systems with Python Version
+  
+  +----------------------------------------------+--------------------------+
```

Comment:
Do we have rendered version for this page? Looks like a formatting issue for this table:

https://github.com/openvinotoolkit/openvino/blob/61ae6ce5517e85e66ff9bc26446fd580fce0bf7a/docs/install_guides/installing-openvino-docker-linux.md

Reply:
http://openvino-docs.inn.intel.com/yuan-CVS-80775/openvino_docs_install_guides_installing_openvino_docker_linux.html  The output is fine. It may be caused by Github not supporting RST table.